tests/test_series.py

```diff
@@ -226,6 +227,21 @@ def test_types_dropna() -> None:
     assert assert_type(s.dropna(axis=0, inplace=True), None) is None
 
 
+def test_pop() -> None:
+    # Testing pop support for hashable types
+    # Due to the bug in https://github.com/pandas-dev/pandas-stubs/issues/627
+    class MyEnum(Enum):
+        FIRST = "tayyar"
+        SECOND = "haydar"
+
+    s = pd.Series([3.2, 4.3], index=[MyEnum.FIRST, MyEnum.SECOND], dtype=float)
+    res = s.pop(MyEnum.FIRST)
+    check(assert_type(res, float), np.float64)
+
+    s2 = pd.Series([3, 5], index=["alibaba", "zuhuratbaba"], dtype=int)
+    check(assert_type(s2.pop("alibaba"), int), np.int64)
```

Comment:
```suggestion
    check(assert_type(s2.pop("alibaba"), int), np.int_)
```
You'll need to use `np.int_` so it passes on Windows.

Reply:
Thanks, I didn't know. I just updated it.